src/VisualStudio/Core/Test.Next/Mocks/TestAnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+
+namespace Roslyn.VisualStudio.Next.UnitTests.Mocks
+{
+    public class TestAnalyzerAssemblyLoader : IAnalyzerAssemblyLoader
```

Comment:
Can you instead use [this](http://source.roslyn.io/#Roslyn.Compilers.UnitTests/AnalyzerFileReferenceTests.cs,282373cf1db99c90,references) test-only loader with identical implementation?